docs/index.rst

```diff
@@ -16,7 +16,7 @@ and constructing genetic designs according to the standardized specifications of
    getting_started
    repositories
    sbol_examples
-   dbtl
```

Comment:
Should dbtl.rst be removed from the repository?

Reply:
Yes, I think so..